mypy/fastparse.py

```diff
@@ -98,21 +98,17 @@ def ast3_parse(source: Union[str, bytes], filename: str, mode: str,
         NamedExpr = Any
         Constant = Any
 except ImportError:
-    if sys.version_info.minor > 2:
```

Comment:
I realized this version check doesn't make much sense anymore, so I just removed it. Everyone getting to importing fastparse should already have checked they are on a compatible version of Python, especially with the extra checks I added to stubgen and dmypy.